{
  "Faults": [
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-12T17:54:00.000+0000",
      "latitude": 51.63018445136843,
      "estimatedRestoration": "2022-04-13T05:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JR6673'",
      "type": "HV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JR6673",
      "reference": "JR6673",
      "name": "HP10 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "HP10 8AA",
        "HP10 8AB",
        "HP10 8AD",
        "HP10 8AF",
        "HP10 8AG",
        "HP10 8AH",
        "HP10 8AR",
        "HP10 8AS",
        "HP10 8DF",
        "HP10 8DJ",
        "HP10 8LB",
        "HP10 8LD",
        "HP10 8LE",
        "HP10 8LF",
        "HP10 8LG",
        "HP10 8LN",
        "HP10 8LQ",
        "HP10 8LU",
        "HP10 8LY",
        "HP10 8LZ",
        "HP10 8NA",
        "HP10 8ND",
        "HP10 8NH",
        "HP10 8NJ",
        "HP10 8NL",
        "HP10 8NU",
        "HP10 8NX",
        "HP10 8PE",
        "HP10 8PF",
        "HP10 8RL",
        "HP15 7PB",
        "HP15 7PE",
        "HP9 1BT",
        "HP9 1JP",
        "HP9 1JR",
        "HP9 1JT",
        "HP9 1JU",
        "HP9 1JY",
        "HP9 1XP",
        "HP9 1XR",
        "HP9 1XS",
        "HP9 1XT",
        "HP9 1XU",
        "HP9 1XW",
        "HP9 1YB"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -0.6826738135477356,
          51.63018445136843
        ],
        "type": "Point"
      },
      "UUID": "6ab67e2f-65a8-4108-a5ea-4ea36617fcc9",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -0.6826738135477356,
      "resolved": false
    },
    {
      "py/object": "Fault",
      "loggedAt": "2022-04-13T03:36:00.000+0000",
      "latitude": 50.81122263494691,
      "estimatedRestoration": "2022-04-13T07:00:00.000+0000",
      "message": "We’re sorry for the loss of supply. We currently have a fault affecting the areas listed. If you need more information, please call us on 105 or send us a message on <a href=\"http://m.me/ssencommunity/\">Facebook</a> or Twitter (<a href=\"https://twitter.com/ssencommunity\">@ssencommunity</a>) and quote reference 'JR6756'",
      "type": "LV",
      "uri": "http://api.sse.com/powerdistribution/network/v3/api/fault/JR6756",
      "reference": "JR6756",
      "name": "SO45 Area",
      "networkId": "com.sse.ssepd.sepd",
      "affectedAreas": [
        "SO45 1YX",
        "SO45 1YY",
        "SO45 1ZN"
      ],
      "location": {
        "__json": true,
        "coordinates": [
          -1.3742789346353959,
          50.81122263494691
        ],
        "type": "Point"
      },
      "estimatedArrivalOnSiteTime": "2022-04-13T05:00:00.000+0000",
      "UUID": "31d7c884-6074-4a3b-80b5-c134d5cdf261",
      "region": "com.opengrid.geo.world.europe.gb",
      "py/class": "http://opengrid.com/givr/1.0#//Fault",
      "longitude": -1.3742789346353959,
      "resolved": false
    }
  ]
}
